hieradata/hosts/mw11.yaml

```diff
@@ -3,7 +3,7 @@ users::groups:
 jobrunner: true
 contactgroups: ['sre', 'mediawiki']
 role::mediawiki::use_strict_firewall: true
-mediawiki::branch: 'REL1_36'
+mediawiki::branch: 'REL1_37'
```

Comment:
Can't this just be removed? It reads from common and to change just test3, it can be explicitly set there. Just makes only having to do one file each time for production upgrades.